homu/server.py

```diff
@@ -117,25 +117,28 @@ def result(repo_label, pull):
 
 @get('/queue/<repo_label:path>')
 def queue(repo_label):
-    if repo_label not in g.cfg['repo'] and repo_label != 'all':
-        abort(404)
+
+    if repo_label == 'all':
+        labels = g.repos.keys()
+    else:
+        labels = repo_label.split('+')
+        if any(label not in g.cfg['repo'] for label in labels):
+            abort(404)
```

Comment:
This suggestion could be applied to  remove the support for combining queues with `+` but keep the `all` queue

```suggestion
    if repo_label == 'all':
        labels = g.repos.keys()
    elif repo_label in g.cfg['repo']:
        labels = [repo_label]
    else:
        abort(404)
```